src/main/java/tc/oc/pgm/listeners/LongRangeTNTListener.java

```diff
@@ -271,7 +272,9 @@ private void removeTNT(Entity entity, boolean explode) {
 
   @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
   public void onPlayerJoin(final PlayerJoinEvent event) {
-    this.viewsByPlayer.put(event.getPlayer(), new View(event.getPlayer()));
+    if (ViaUtils.getProtocolVersion(event.getPlayer()) >= 47) {
```

Comment:
Create a util for protocol versions, `47` means nothing but `Protocol.VERSION_1_7` (or something along those lines) is easy to read